Add a reasonable "getting started" guide to the user docs

matentzn opened the pull request:
Fixes #435 

- [x] `docs/` have been added/updated if necessary
- [ ] `make test` has been run locally
- [ ] tests have been added/updated (if applicable)
- [ ] [CHANGELOG.md](https://github.com/mapping-commons/sssom/blob/master/CHANGELOG.md) has been updated.

If you are proposing a change to the SSSOM metadata model, you must 

- [ ] provide a full, working and valid example in `examples/`
- [ ] provide a link to the related GitHub issue in the `see_also` field of the linkml model
- [ ] provide a link to a valid example in the `see_also` field of the linkml model
- [ ] make sure any new slot is annotated with the appropriate `added_in` annotation
- [ ] update the "Model changes across versions" (in `src/docs/spec-models.md`) accordingly
- [ ] run SSSOM-Py test suite against the updated model

matentzn committed:
Add getting started guide stub to the user docs

matentzn committed:
Merge branch 'master' into getting-started

matentzn committed:
Update getting started guide

matentzn committed:
Merge branch 'master' into getting-started

gouttegd reviewed (dismissed):
No objection but there are still a few things that in my not-so-humble-opinion would deserve some changes.

matentzn committed:
Address review feedback on getting-started guide

- Tone down spreadsheet praise; acknowledge validation trade-offs and link to discussion
- State that CURIEs are required by the SSSOM/TSV format (not just suggested); remove the `https: https` URI hack
- Remove recommendation of owl:equivalentClass and rdfs:subClassOf as mapping predicates
- Add references to sssom-java's sssom-cli tool in validation and conversion sections

matentzn committed:
Update src/docs/getting-started.md

Co-authored-by: Charles Tapley Hoyt <cthoyt@gmail.com>

matentzn commented on src/docs/getting-started.md:
```diff
@@ -0,0 +1,265 @@
+
+For a more detailed walkthrough, see the [SSSOM Toolkit guide](toolkit.md) and the [sssom-py documentation](https://mapping-commons.github.io/sssom-py).
+
+#### Using SSSOM with the Ontology Access Kit (OAK)
```
You are right I will remove it.

matentzn committed:
Address review feedback getting-started

- Replace fictional Kewl Foodie example with real MP-HP mappings from uPheno
- Use copy-pastable validation example with biomappings
- Remove standalone OAK section (avoid singling out tools)
- Add tutorial link to ODK section
- run "prettier"

matentzn committed:
Merge branch 'master' into getting-started

cthoyt reviewed (dismissed):
lets merge this then do addition rounds of PR for improvements / other comments

matentzn committed:
Delete commit.txt

matentzn committed:
Merge branch 'master' into getting-started
